src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -0,0 +1,247 @@
+
+1. After the `ceph osd reweight` command completes, verify that Ceph is in a healthy state:
+
+   ```
+   ceph status
+   ```
+
+1. On the first node that you intend to remove, run the following ceph CLI command to mark the OSD as `down`:
+
+   ```
+   ceph osd down OSD_NAME
+   ```
+
+1. On the first node that you intend to remove, use the kubectl command-line tool to scale the OSD deployments to 0 replicas:
+
+   ```
+   kubectl scale deployment -n rook-ceph rook-ceph-osd-1 --replicas 0
+   ```   
+
+1. Remove the node.
+
+   After the node is removed, Ceph replicates its data to OSDs on remaining nodes. If a Ceph OSD is scheduled on the node that you removed, verify that Ceph is in a healthy state.
+
+1. (Optional) After the node is removed, you can run the kURL reset script to remove kURL and Kubernetes assets from the node to prep it to re-join the cluster at a later time. Or, delete the VM and provision a new VM.
+
+Repeat the steps in this procedure for any remaining nodes that you want to remove. Verify that Ceph is in a healthy state before you remove any additional nodes.
+
+### Remove Nodes with ECKO
+
+You can use EKCO add-on scripts to programmatically cordon and purge a node so that you can then remove the node from the cluster.
+
+_**Warnings**_: Consider the following warnings about data loss before you proceed with this procedure:
+
+    * **Ceph health**: The EKCO scripts in this procedure provide a quick method for cordoning a node and purging Ceph OSDs so that you can remove the node. This procedure is _not_ recommended unless you are able to confirm that Ceph is in a healthy state. If Ceph is not in a healthy state before you remove a node, you risk data loss.
+
+       To verify that Ceph is in a healthy state, run the following `ceph status` command in the `rook-ceph-tools` or `rook-ceph-operator` Pod in the `rook-ceph` namespace for Rook Ceph v1.4 or later:
+
+       ```
+       kubectl -n rook-ceph exec deploy/rook-ceph-tools -- ceph status
+       ```
+
+    * **Data replication**: A common Ceph configuration is three data replicas across three Ceph OSDs. It is possible for Ceph to report a healthy status without data being replicated properly across all OSDs. For example, in a single-node cluster, there are not multiple machines where Ceph can replicate data. In this case, even if Ceph reports healthy, removing a node will result in data loss because the data was not properly replicated across multiple OSDs on multiple machines.
+
+    If you are not certain that Ceph data replication was configured and completed properly, or if Ceph is not in a healthy state, it is recommended that you instead follow the procedure to first rebalance the data off the node that you intend to remove to avoid data loss. See [(Recommended) Manually Rebalance Ceph and Remove a Node](#ecommended-manually-rebalance-ceph-and-remove-a-node) above.
```

Comment:
s/ it is recommended that you instead follow the procedure to first rebalance the data off the node that you intend to remove to avoid data loss / it is recommended that you first rebalance the data off the node that you intend to remove to avoid data loss